wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md

```diff
@@ -183,6 +184,10 @@ In stable, slider ends would play their hitsounds even if they were missed, as l
 
 Instead of a speed cap, spinners now have a score cap. This was done as a simple initial implementation which may be revised in the future.
 
+The score cap is determined by the total score that can be obtained by spinning the entire spinner with 477 RPM.
+
+That means you can spin the entire spinner at 477 RPM to get the highest score, or spin at a higher speed to finish spinner early, but you can still only get the score which spin the entire spinner at 477 RPM.
```

Comment:
currently it's more like 480 RPM (8 RPS) until https://github.com/ppy/osu/pull/24661 is merged

also it's really a rotation cap

```suggestion
Instead of a speed cap, spinners now have a rotation cap determined by the total amount of rotations that can be achieved by spinning the entire spinner at 477 RPM.

This means that the maximum score can be obtained early by spinning faster, with no more points awarded afterwards for the rest of the spinner duration.
```